crates/apub/src/context.rs

```diff
@@ -25,3 +25,19 @@ impl<T> WithContext<T> {
     self.inner
   }
 }
+
+#[derive(Serialize, Deserialize)]
+pub(crate) struct WithContextJson {
+  #[serde(rename = "@context")]
+  context: OneOrMany<AnyBase>,
+  inner: serde_json::Value,
```

Comment:
Just noticed you forgot `#[serde(flatten)]` here.

Reply:
Actually, the whole thing doesnt make sense because the activity is already stored in the database with context. So I think we just need to remove the `WithContextJson::new()` from create_json_apub_response().

Reply:
flatten was the source of the error, its the only difference between these. You can only flatten maps and structs, not json.

Reply:
Okay i will fix it.

Reply:
My fix wasnt working either (i simply removed all `\` from the string, and removed the `"` at the beginning and end). Messed the content field which contains escaped html.

Reply:
I don't understand, this PR did fix that problem. Removing flatten for `serde::Value` was all it took.